alicja/config/proto/interface.proto

```diff
@@ -9,13 +9,13 @@ option go_package = "github.com/googleinterns/step224-2020/config/proto"
 import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
 
 service Hermes_Ceph {
```

Comment:
Add a comment above with a description of what this service's responsibilities are and the what it's methods allow you to do. e.g. Include something along the lines of "provides methods to communicate instructions to Cloudprober regarding file operations".